### PR TITLE
Add style for non-overlapping scrollbar in IE

### LINKED ISSUE #24057: Horizontal scrollbar obstructs Bootstrap CDN links in ( Windows 7 ) Internet Explorer Version 11.0.9600.18792..
The page

http://getbootstrap.com/docs/4.0/getting-started/download/

In ( Windows 7 ) Internet Explorer Version 11.0.9600.18792

Seems to add a horizontal scroll bar to the Bootstrap CDN sections that covers the lower part of the links shown when I give them focus (to copy them).

### PR DESCRIPTION
**Pull request by prateekgoel**
Added `-ms-overflow-style: scrollbar;` to resolve overlapping scrollbar issue in IE 11 and Edge.

Fixes #24057

**Before**
image

**After**
image

### CODE REVIEW

**Comment by Johann-S** [comment]
Not sure if this change should be applied to Bootstrap or to our docs CSS :thinking:

**Comment by XhmikosR** [comment]
Does the issue happen in all `pre`s? If so I guess it makes sense to add it in reboot otherwise only docs.

**Comment by prateekgoel** [comment]
Yes, it happens in all.
It's safe to add in reboot.

**Comment by XhmikosR** [comment]
@Johann-S: can you confirm?

**Comment by Johann-S** [comment]
yep I reproduced that bug on Win7 IE 11, but to see it you have to move your cursor over our `<pre>` and it's only for long `<pre>`s

**Review by XhmikosR — APPROVED** [review]

File: scss/_reboot.scss [hunk 1]
```diff
@@ -239,6 +239,9 @@ pre {
   margin-bottom: 1rem;
   // Don't allow content to break outside
   overflow: auto;
+  // We have @viewport set which causes scrollbars to overlap content in IE11 and Edge, so
+  //    we force a non-overlapping, non-auto-hiding scrollbar to counteract.
```
Comment (mdo) [inline scss/_reboot.scss]: Too many spaces here between `//` and `we`.

**Review by mdo — APPROVED** [review]
Good to go other than this one formatting comment...

**Comment by XhmikosR** [comment]
@prateekgoel: please don't remove the option to allow project members to edit your PRs.

**Comment by prateekgoel** [comment]
Yes @XhmikosR . I mistakenly unchecked that option.
I have removed the extra spaces as mentioned by @mdo.

**Pull request merged**